### PR TITLE
[TT-7372] Add 1.18 and 1.19

### PR DESCRIPTION
**Pull request by titpetric**
[changelog]
added: go 1.18 and 1.19.

### CODE REVIEW

**Comment by titpetric** [comment]
- changed CI to split buster and stretch builds,
- stretch builds from 1.15-1.18 (deprecated going forward)
- buster builds from 1.15-1.19 (no official 1.20 release yet)

**Review by alephnull — COMMENTED** [review]
Who is using 1.17? And 1.18?

**Comment by alephnull** [comment]
Also, stretch is required for RHEL7 support. If we don't build a strech build env, RHEL7 will not be supported. This is a decision that product and Leo need to sign off on.

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -13,11 +13,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        gover: [ 1.15, 1.16, 1.17 ]
-        debver: [ buster, stretch ]
-        include:
-          - debver: stretch
-            tagSuffix: -el7
+        gover: [ 1.15, 1.16, 1.17, 1.18, 1.19 ]
+        debver: [ buster ]
```
[the remark this answers is not in the document]
Comment (ermirizio) [reply .github/workflows/release.yml:17]: Also what did motivates you to split the CI in two different workflows? I'm not seeing the benefit, but duplicated code

**Comment by titpetric** [comment]
> Who is using 1.17? And 1.18?

The versions have been added for completeness. As RHEL is a concern, we may want to build our stretch env, but stretch is also EOL upstream in 6 months. I've cc'd Leo and will explicitly raise the concern before we act on any upgrades, of course.

**Comment by alephnull** [comment]
@ermirizio Would you please review this one?

**Comment by titpetric** [comment]
@alephnull - this includes an additional environment bump now (from buster to bullseye). I asume everything will work out, but it's an additional environment change. Since 1.19 is supported on buster, and we do run buster in CI, should we make the upgrade from buster->bullseye a separate concern?

**Comment by alephnull** [comment]
@titpetric I'll leave it up to you. Buster is actually quite old.

**Comment by titpetric** [comment]
Buster is a tested platform today and it is supported until mid 2024 it seems. Given current release practices, we could test both debian release images - bullseye, buster), as we already do it twice today for buster and stretch(rh7).

For that, we need a buster 1.19 image available along with a bullseye 1.19.

**Review by alephnull — APPROVED** [review]
LGTM :rocket:

**Pull request merged**